Source/SwiftLintFramework/Extensions/File+SwiftLint.swift

```diff
@@ -181,4 +181,31 @@ extension File {
         }
         return violatingRanges
     }
+
+    internal func numberOfCommentOnlyLines(startLine: Int, endLine: Int) -> Int {
```

Comment:
I thought these functions also excluded whitespace only lines? It doesn't seem like they do. I'm not sure what I'm missing.